docs/src/modules/reference/pages/release-notes.adoc

```diff
@@ -12,6 +12,21 @@ Current versions
 
 == February 2025
 
+* https://github.com/akka/akka-sdk/releases/tag/v3.2.0[Akka SDK 3.2.0]
+    - gRPC endpoint support
+    - APIs for serving static resources from HTTP endpoints
+    - API for producing Server Sent Events (SSE) from HTTP endpoints
+    - Wildcard paths for HTTP endpoints
+
+* Akka Runtime 1.3.6
+    - View query parameter improvements
+    - Wildcard path support
+
+* Akka Runtime 1.2.9
+    - View query parameter improvements
```

Comment:
Wondering if we need/want the 1.2.* series here or only on the Kalix docs..

Reply:
The idea is to have one single aggregated release feed. That's why we also include releases of libraries here.

Reply:
Ah I see. 👍